repair_scheduled_date_calendar_view/readme/USAGE.md

```
@@ -0,0 +1,3 @@
+Use the calendar view to see the scheduled repair orders.
+You can specify duration of any specific order by modifying
+the "Planned Duration" field or by resizing the calendar element.
```

markdownlint-cli2 (0.18.1)

Files should end with a single newline character

(MD047, single-trailing-newline)